llvm/include/llvm/BinaryFormat/ELF.h

```diff
@@ -1542,6 +1542,13 @@ enum {
   NT_AMDGPU_METADATA = 32
 };
 
+// AMD LLPC specific notes.
+enum {
+  // Note types with values between 0 and 32 (inclusive) are reserved.
+  NT_AMD_LLPC_CACHE_HASH = 33,
```

Comment:
I think these new ones should be in the same space as the code object V3 ones, so in the enum above and called NT_AMDGPU_*. I have made that modification to your change when pushing it for internal review, so no need for you to change it here.

So I was wrong that the vendor name is "AMD". It is "AMDGPU".

Reply:
Actually, could you please revise this change to put them in the code object V3 enum and name them NT_AMDGPU_*? I realized I would have to change the tests too (moving the new ones into a different test), and I don't really have time to start on that now.

Reply:
done.